docs/tutorial/Pre-requisites of running a Waku node.md

```diff
@@ -0,0 +1,116 @@
+---
+title: 'Pre-requisites of running a Waku node'
+---
+
+Pre-requisites of running a Waku node
+===
+
+## Table of Contents
+
+[TOC]
+
+## Creating Goerli Ethereum Account and obtaining its private key
+
+
+1. Download and install Metamask [https://metamask.io/download/](https://metamask.io/download/)
+2. Create a new wallet and save your secret recovery key
+    
+    ![](https://i.imgur.com/HEOI0kp.jpg)
+
+3. Login to Metamask
+    
+    ![](https://i.imgur.com/zFduIV8.jpg)
+
+4. You should already see an account created. As you can see on top right, it should be pointing to Ethereum mainnet.
+    
+    ![](https://i.imgur.com/gk3TWUd.jpg)
+
+5. You can use the same account for different networks.
+6. You can switch to a test network by clicking on the drop down menu like so:
```

Comment:
As per our 1:1, please revise these parts to be focused on what is needed for rln-relay dogfooding, no need for the general cases.